src/pages/RepoPage/FailedTestsTab/FailedTestsTable/FailedTestsTable.tsx

```diff
@@ -0,0 +1,253 @@
+import type { SortingState } from '@tanstack/react-table'
+import {
+  createColumnHelper,
+  flexRender,
+  getCoreRowModel,
+  getSortedRowModel,
+  useReactTable,
+} from '@tanstack/react-table'
+import cs from 'classnames'
+import { useState } from 'react'
+// import { useInView } from 'react-intersection-observer'
+// import { useParams } from 'react-router-dom'
+
+import { OrderingDirection } from 'services/repos'
+import Icon from 'ui/Icon'
+import Spinner from 'ui/Spinner'
+
+const Loader = () => (
+  <div className="mt-16 flex flex-1 items-center justify-center">
+    <Spinner />
+  </div>
+)
+
+// function LoadMoreTrigger({
+//   intersectionRef,
+// }: {
+//   intersectionRef: React.Ref<HTMLSpanElement>
+// }) {
+//   return (
+//     <span
+//       ref={intersectionRef}
+//       className="invisible relative top-[-65px] block leading-[0]"
+//     >
+//       Loading
+//     </span>
+//   )
+// }
+
+export function getSortingOption(
+  sorting: Array<{ id: string; desc: boolean }>
+) {
+  const state = sorting[0]
+
+  if (state) {
+    const direction = state.desc
+      ? OrderingDirection.DESC
+      : OrderingDirection.ASC
+
+    let ordering
+    if (state.id === 'testName') {
+      ordering = 'NAME'
+    }
+
+    if (state.id === 'lastDuration') {
+      ordering = 'LAST DURATION'
+    }
+
+    if (state.id === 'failureRate') {
+      ordering = 'FAILURE RATE'
+    }
+
+    if (state.id === 'commitsFailed') {
+      ordering = 'COMMITS FAILED'
+    }
+
+    if (state.id === 'lastRun') {
+      ordering = 'LAST RUN'
+    }
+
+    return { direction, ordering }
+  }
+
+  return undefined
+}
+
+interface FailedTestsColumns {
+  testName: string
```

Comment:
`name` instead of `testName`

Reply:
Updated in https://github.com/codecov/gazebo/pull/3042/commits/18984d4a734599b3a37b2fc89de0d9a269d73d90